contracts/Sale.sol

```diff
@@ -216,6 +222,15 @@ contract Sale {
         owner = _newOwner;
     }
 
+    function withdrawRemainder()
+        onlyOwner
+        saleStarted
```

Comment:
This seems unnecessary, no?

Reply:
The reason I put that was to ward off any funny business of the `endBlock` being earlier than the start block. I suppose there would be no money if that's the case though :P